### ISSUE TITLE
[Tabs] DAP violation on selected tab

### ISSUE THREAD

**Issue by Tony9984** [carbon-design-system/carbon#6113]
## Environment

> Operating system

MacOS Catalina

> Browser

Chrome

> Assistive technology used to verify

DAP

## Detailed description

> What did you expect to happen?

no DAP violations on components

> What happened instead?

got a violation of "Multiple 'navigation' sections or landmarks must have unique labels specified with 'aria-label' or 'aria-labelledby'" on the tab I am currently on

## Steps to reproduce the issue

1. Create a few Tab components nested inside a Tabs component
2. Select an arbitrary tab
3. Run the DAP extension on chrome

## Additional information

![image](https://user-images.githubusercontent.com/36376980/81714125-babf2f00-9444-11ea-847a-b85476d49a9c.png)

**Comment by dakahn** [comment]
So like the error explains, each navigation region on your page needs a unique name. You can provide that name via aria-label and the violation should resolve. Closing, but happy to reopen if you're still having trouble.
